app/controllers/users_controller.rb

```diff
@@ -7,4 +7,17 @@ def show
     @user = User.find_by_id(params[:id])
     @posts = @user.most_recent_posts
   end
+
+  # def new
+  #  @user = User.new
+  # end
+
+  # def create
+  #  @user = User.create(params[:user])
+  #  if @user.save
+  #    redirect_to user_path(@user.id)
+  #  else
+  #    render :new
+  #  end
+  # end
```

Comment:
### Optional
- [ ] It would be nice if we remove the commented piece of code, it makes our code base looks messy and difficult to understand